pkg/apis/options/sessions.go

```diff
@@ -1,10 +1,13 @@
 package options
 
+import "time"
+
 // SessionOptions contains configuration options for the SessionStore providers.
 type SessionOptions struct {
-	Type   string             `flag:"session-store-type" cfg:"session_store_type"`
-	Cookie CookieStoreOptions `cfg:",squash"`
-	Redis  RedisStoreOptions  `cfg:",squash"`
+	Type                string             `flag:"session-store-type" cfg:"session_store_type"`
+	SessionExpiryOffset time.Duration      `flag:"session-expiry-offset" cfg:"session_expiry_offset"`
```

Comment:
This will already have session in the name typically, so maybe we can call this just `ExpiryOffset`?